deploy/webhook.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: webhook-webhook-service
```

Comment:
Can this name be made unique since there are other controllers running the same namespace? We don't want conflicts.

Reply:
How about propagator-webhook-service